conans/test/functional/build_helpers/visual_environment_test.py

```diff
@@ -154,12 +154,12 @@ def build(self):
 
         result = {"Debug": "['-Zi', '-Ob0', '-Od']",
                   "Release": "['-DNDEBUG', '-O2', '-Ob2']",
-                  "RelWithDebInfo": "['-Zi', '-O2', '-Ob1']",
-                  "MinSizeRel": "['-O1', '-Ob1']"}
+                  "RelWithDebInfo": "['-DNDEBUG', '-Zi', '-O2', '-Ob1']",
```

Comment:
If this was a bug, it is better to extract it and do a separate PR, so we can make sure to fix it, independently of new features like this about the Intel compiler.

Reply:
No need to do it now, just a suggestion for the future

Reply:
okay, thanks!